src/blockjacobi.jl

```diff
@@ -0,0 +1,60 @@
+using SuiteSparse
+using SparseArrays
+using LinearAlgebra
+mutable struct BlockJacobi{T, S<:AbstractMatrix{T}} <: LSPreconditioners.Preconditioner
+    nblocks :: Int64
+    blocksizes :: Array{Int32}
+    blocks :: Vector{Union{LU{T, Matrix{T},Vector{Int32}},SuiteSparse.UMFPACK.UmfpackLU{Float64,Int64}}}
+end
+
+# Function to form a block Jacobi preconditioner
+function BlockJacobi(A::Matrix,blocksize::Integer)
+    m = size(A,1)
```

Comment:
@npritch928 space after A. ditto in the following ones. Please refer to style guides, code , etc.